Imaginary_tvOSTests/Imaginary_tvOSTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  Imaginary_tvOSTests.swift
+//  Imaginary_tvOSTests
+//
+//  Created by Khoa Pham on 22.09.2017.
+//  Copyright © 2017 Hyper Interaktiv AS. All rights reserved.
+//
+
+import XCTest
+
+class Imaginary_tvOSTests: XCTestCase {
+    
```

Comment:
If we don't have tests I would remove this.

Reply:
👍